jupyterlab-dataops/resources/cloudbeaver/conf/initial-data-sources.conf

```diff
@@ -0,0 +1,24 @@
+{
+	"folders": {},
+	"connections": {
+		"google_bigquery_jdbc_simba-1": {
+			"provider": "bigquery",
+			"driver": "google_bigquery_jdbc_simba",
+			"name": "Google BigQuery",
+			"save-password": true,
+			"show-system-objects": true,
+			"read-only": false,
+			"configuration": {
+				"host": "https://www.googleapis.com/bigquery/v2",
```

Comment:
same here, we are hardcoding GCP, do we want it?

Reply:
this endpoint stays the same AFAIK for bigquery connection

Reply:
It serves only as template connection though. The connection itself should be configured in some external by uploading/setting service account JSON. We could get rid of this configuration here, no?